docs/custom-workload.md

```diff
@@ -0,0 +1,92 @@
+    matchLabels:
+      app: win-webserver
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: win-webserver
+      name: win-webserver
+    spec:
+      tolerations:
+      - key: "os"
+        value: "Windows"
+        Effect: "NoSchedule"
+      containers:
+      - name: windowswebserver
+        image: mcr.microsoft.com/windows/servercore:ltsc2019
+        imagePullPolicy: IfNotPresent
+        command:
+        - powershell.exe
+        - -command
+        - $listener = New-Object System.Net.HttpListener; $listener.Prefixes.Add('http://*:80/'); $listener.Start();Write-Host('Listening at http://*:80/'); while ($listener.IsListening) { $context = $listener.GetContext(); $response = $context.Response; $content='<html><body><H1>Red Hat OpenShift + Windows Container Workloads</H1></body></html>'; $buffer = [System.Text.Encoding]::UTF8.GetBytes($content); $response.ContentLength64 = $buffer.Length; $response.OutputStream.Write($buffer, 0, $buffer.Length); $response.Close(); };
+        securityContext:
+          runAsNonRoot: false
+          windowsOptions:
+            runAsUserName: "ContainerAdministrator"
+      nodeSelector:
+        kubernetes.io/os: windows
+```
+
+For more information about windows workloads, visit
+[the docs](<https://docs.openshift.com/container-platform/4.12/windows_containers/scheduling-windows-workloads.html>).
+
+## Configure the base image
+
+It is very important to set the correct base image version for your test workload.
+Each version of the WMCO uses a specific windows operating system version, and each workload has its own base image.
+
+To know which windows version your release uses, check the defaultSKU in
+[azure.go](<https://github.com/openshift/windows-machine-config-operator/blob/master/test/e2e/providers/azure/azure.go#L21>)
+for the WMCO release you’re testing with.
+To know which base image your release uses, check your Machineset.yaml in the repo root, or your BYOH image. 
+
+### Check the operating system on the cluster
+
+To be sure that your deployment’s base image corresponds with the cluster’s OS version, you should check what version
+of OS your cluster is running.
+
+### Edit your deployment
+
+Now that you know what operating system is running, you have to find the corresponding base image to set your
+deployment to. This is an incomplete table of which windows version corresponds with which powershell base image.
+
+| Windows Version                 | Base Image version                                   |
+|---------------------------------|------------------------------------------------------|
+| 2019-Datacenter-with-Containers | mcr.microsoft.com/powershell:lts-nanoserver-1809     |
+| 2022-datacenter-smalldisk       | mcr.microsoft.com/powershell:lts-nanoserver-ltsc2022 |
+
+### Set the powershell executable
+
+Different base images have different executables for powershell. Since WMCO uses the powershell nanoserver images, the
+command should be set to pwsh.exe.
```

Comment:
Again I don't think this should live in HACKING.md. These are troubleshooting steps